docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,43 @@
+| [Multik](https://github.com/Kotlin/multik)                                   | - Data cleaning and processing  <br/> - Model building                                                                   | - Mathematical operations over multidimensional arrays (linear algebra, statistics, arithmetics, and other calculations) <br/> - Creating, copying, indexing, slicing, and other array operations <br/> - Kotlin-idiomatic library with benefits such as type and dimension safety and swappable computational engines, running on the JVM or as native code                                                                                                                                                                                                                                                                                                     |
+| [KMath](https://github.com/mipt-npm/kmath)                                   | - Data cleaning and processing <br/> - Data exploration and visualization <br/> - Model building                         | - Modular library to work with mathematical abstractions in [Kotlin Multiplatform](https://www.jetbrains.com/kotlin-multiplatform/) (JVM, JS, Native, and Wasm) <br/> - APIs for algebraic structures, mathematical expressions, histograms, and streaming operations <br/> - Interchangeable wrappers over existing Java and Kotlin libraries, including [ND4J](https://github.com/eclipse/deeplearning4j/tree/master/nd4j), [Apache Commons Math](https://commons.apache.org/proper/commons-math/), [Multik](https://github.com/Kotlin/multik), and others <br/> - Inspired by Python's [NumPy](https://numpy.org/) but allowing superior architecture designs |
+| [londogard-nlp-toolkit](https://github.com/londogard/londogard-nlp-toolkit/) | - Data cleaning and processing <br/> - Model building                                                                    | - Utilities for natural language processing <br/> - Library written in Kotlin for the JVM <br/> - Word embeddings, stopwords, stemming, tokenizers, vectorizers, and more processing tools <br/> - Ready-made algorithms for machine learning and deep learning                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Note: The last code update for this library was in 2022

Reply:
Remove please londogard